packages/lib/display/service.ts

```diff
@@ -236,6 +236,12 @@ export const getDisplayCountBySurveyId = reactCache(
                     lte: filters.createdAt.max,
                   },
                 }),
+              ...(filters &&
+                filters.responseIds && {
+                  responseId: {
+                    in: filters.responseIds,
```

Comment:
Since we don't have an issue on the responseId I think this might give performance issues (we usually have a huge amount of displays in the table on Formbricks Cloud). Please check back with chatGPT if we need to update our indexes in the `Display` model.

Reply:
@mattinannt we can skip this step if we modify the display model schema and create a link of display to the response
![image](https://github.com/user-attachments/assets/f6d071bb-92eb-4d84-a727-6dcf864d5a69)